trends_presentation/src/commonTest/kotlin/net/thechance/mena/trends/presentation/screen/manage_my_trends/ManageTrendsViewModelTest.kt

```diff
@@ -94,6 +96,20 @@ class ManageTrendsViewModelTest {
         }
     }
 
+    @Test
+    fun `onRetryClick should reset error and call getFeedReels`() = runTest {
```

Comment:
The name of function need to include all it do( get current user info and trends) so it will become :
```suggestion
    fun `onRetryClick should reset error and call the both function getFeedReels and getCurrentUserInfo `() = runTest {
```
Or any name you see for including the both function